CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.5.5] - 2020-10-23
+
+### Fixed
+- Incorrect example for creating relationships
```

Comment:
Given that this is a docs change, I don't think it should be it in the change log.